pkg/controller/nodes/task/sleeper_plugin.go

```diff
@@ -0,0 +1,61 @@
+package task
```

Comment:
nit:

```suggestion
// +build test_tools

package task
```

build with the plugin:
`go build -tags test_tools`